### PR TITLE
Initial Scaffold

### LINKED ISSUE #5: Scafolding for time track
A model for time track:

-response_map_id (int) This should be automatically created when ref to resposne_map

-time_data(string)

### PR DESCRIPTION
**Pull request by boergerj**
Contains initial scaffold for ReviewTimes

- Includes model and controller
- Removes views
- Removes "new" method from controller

### CODE REVIEW

**Review by Hatsukikeika — APPROVED** [review]
Approved

File: db/schema.rb [hunk 1]
```diff
@@ -742,14 +740,14 @@
...
-    t.datetime "created_at",                null: false
-    t.datetime "updated_at",                null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.integer  "notification_id", limit: 4, null: false
   end
 
-  add_index "track_notifications", ["notification_id"], name: "index_track_notifications_on_notification_id", using: :btree
-  add_index "track_notifications", ["user_id"], name: "index_track_notifications_on_user_id", using: :btree
+  add_index "track_notifications", ["notification_id"], name: "notification_id", using: :btree
```
Comment (AvlWx2014) [inline db/schema.rb:749]: It's weird that the scaffold would have changed this stuff, since it seems unrelated. We should probably figure out why, so when we're inevitably asked we can explain why.

**Review by AvlWx2014 — COMMENTED** [review]
After our last conversation with Kai, and after reading some of the 2080 implementation, should we include a column for review round in the database? If we're stuffing all the timing data in JSON in the `time_data` column, then we can get away with having an entry per review round. And it sounds like we will need to be able to break the timing data down by round.

**Comment by boergerj** [comment]
Not needed anymore using 2080 code as base
